elyra/cli/pipeline_app.py

```diff
@@ -512,8 +513,19 @@ def describe(json_option, pipeline_path):
     indent_length = 4
     blank_field = "Not Specified"
     blank_list = ["None Listed"]
-    pipeline_keys = ["name", "description", "type", "version", "nodes", "file_dependencies", "component_dependencies"]
-    iter_keys = {"file_dependencies", "component_dependencies"}
+    pipeline_keys = [
+        "name",
+        "description",
+        "type",
+        "version",
+        "nodes",
+        "file_dependencies",
+        "component_dependencies",
+        "mounted_volumes",
```

Comment:
Use the constant value for `mounted_volumes` as defined in https://github.com/elyra-ai/elyra/blob/master/elyra/pipeline/pipeline_constants.py#L20. (See other comment, which includes a clarification what these constants were introduced for)

Reply:
@ptitzler , just one doubt. 
For referencing pipeline dictionary keys, I am replacing the key "mounted_volumes" with the corresponding constant. This is clear.
But for list, set initialization also, should we be doing this ? Rechecking my understanding since the list is local and technically may have any names.
Kindly let me know. Thnx

Reply:
Good question! The `describe_dict` keys (e.g. `describe_dict["component_dependencies"]`) should remain as is for now because their names are used to produce the output (`Component Dependencies: ...`) and should remain consistent over time. For example, let's say we decide in the future to rename the value of `MOUNTED_VOLUMES` in the pipeline constants file from `component_dependencies` to `custom_component_dependencies`. We don't necessarily want this internal name change to  surface to the user.  If we were to use the constant value, we always would and lose a layer of abstraction that is useful.

There's definitely room for improvement in the describe method code to implement this more cleanly -- e.g. each dict entry gets assigned a value for a human-readable display name (`Component dependencies`) and a machine-readable property name (`custom_components`) that is used when the output is JSON formatted).

Reply:
Fixed as discussed. Removed usage of iter_keys and pipeline_keys and used describe_dict straight away for producing JSON and non-JSON output